src/gate/library/liberty/translate.cpp

```diff
@@ -33,7 +33,7 @@ static void fillInputs(
     GModel::GNet &net,
     std::map<size_t, GateId> &inputs) {
   std::vector<size_t> tmp;
-  for (auto [name, wire]: wires) {
+  for (auto &[name, wire]: wires) {
```

Comment:
utopia-eda/src/gate/library/liberty/translate.cpp:36:8: warning: 'auto &' can be declared as 'const auto &' [llvm-qualified-auto]